Sources/MongoSwift/BSON/BSONDecoder.swift

```diff
@@ -2,8 +2,6 @@ import Foundation
 
 /// `BSONDecoder` facilitates the decoding of BSON into semantic `Decodable` types.
 public class BSONDecoder {
-    // swiftlint:disable explicit_acl
```

Comment:
I think the version of swiftlint on travis is still broken for this rule so let's leave it here for now